src/main/java/com/xeeva/catalog/ItemDetailsPage.java

```diff
@@ -140,13 +147,22 @@ public void modifyItemDetails(String UnitPrice, String Quantity, String strUOMVa
 	 **/
 	public void add_TwoDiffrent_ItemsToCart(String UP,String Qty,String UOM){
 		TestReporter.logStep("UOM "+lstUOMGlobalCart.size());
-		TestReporter.logStep("Add To Cart buttons "+btnIAddToCartItems.size());
+		TestReporter.logStep("Add To Cart Items : "+btnIAddToCartItems.size());
+		driver.setPageTimeout(10, TimeUnit.SECONDS);
 		lstUOMGlobalCart.get(0).select(UOM);
 		driver.executeJavaScript("arguments[0].click();", btnIAddToCartItems.get(0));
-		driver.setPageTimeout(5);
+		//Handle Alert if present
+		  if(AlertHandler.isAlertPresent(driver, 6)){
+		   AlertHandler.handleAlert(driver, 6);
+		  }
+		driver.setPageTimeout(5, TimeUnit.SECONDS);
 		lstUOMGlobalCart.get(1).select(UOM);
 		driver.executeJavaScript("arguments[0].click();", btnIAddToCartItems.get(1));
-		driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);

```

Comment:
Please remove - again - the implicitlyWait (same as driver.setElementTimeout()), only changes timeout if you are using driver.findElement methods